daemon/daemon.go

```diff
@@ -107,11 +110,16 @@ func (d *daemon) LoadConfig(newConfig config.Config) error {
 		return fmt.Errorf("unrecognized handler %s", d.proxy.Handler)
 	}
 
-	middleware.EnableRollups(context.Background(), logger.New("sphinx"), 10*time.Second)
+	middleware.EnableRollups(context.Background(), logger.New("sphinx"), 20*time.Second)
 	d.handler = middleware.New(handler, "sphinx", func(req *http.Request) map[string]interface{} {
+		path := req.URL.Path
+		matches := pathRegex.FindAllStringSubmatch(path, 1)
+		if len(matches) == 1 && len(matches[0]) == 4 {
+			path = matches[0][1] + matches[0][3]
+		}
```

Comment:
Please add some comments for this magic

I personally find it hard to understand which of `(/.*)(/[0-9a-f]{24})(.*)` matches 0,1,2,3